apps/studio/prisma/schema.prisma

```diff
@@ -75,6 +96,7 @@ model User {
 
   permission  Permission[]
   siteMembers SiteMember[]
+  Version     Version[]
```

Comment:
nit: lowerCamelCase

```suggestion
  version     Version[]
```

Also does it make sense for this (plus the siteMembers/permission) to be optional?

Reply:
All these are just relations

Or do you mean the user is optional within the Permission and SiteMember models? But both perm and sitemember needs to be associated to an user right?

Reply:
Hmm in this case it means that a user would definitely have permission + site membership, but this might not be entirely true especially for a user that is just created/first log in.

Reply:
Clarified offline, no need for `undefined` here because it is captured inside the empty array `[]`.